### DISCUSSION TITLE
Filtering the products

### DISCUSSION THREAD

**Discussion by asadqureshi315** [medusajs/medusa#4951] [Q&A]
hey, I am new to Medusa and was trying to find any existing plugins which can help with filtering.
I want to filter data availbale in store front, how can i do it

**Comment by tsvetann** [comment]
Meilisearch/algolia?

**Reply by asadqureshi315** [reply]
I used algolia and it works perfectly and i also created multiple collections to get products based on collections, but i want to see if there is any filtering or sorting features which alllow us to find products based on cost such as >10$ or <100$

**Reply by tsvetann** [reply]
Unfortunately not that I have discovered.
